buildsystem/deception-env.sh

```diff
@@ -1,4 +1,4 @@
-#!/bin/bash
+source /etc/profile.d/modules.sh
```

Comment:
```suggestion
#!/bin/bash

source /etc/profile.d/modules.sh
```

Don't know why this was nuked - you still need bash to run module commands and scripts...